### ISSUE TITLE
Even if no connection to other devices, Battery consumtaiton is 20 %

### ISSUE THREAD

**Issue by Mannshoch** [syncthing/syncthing-android#942]
I detected that  in the battery overview the application called "Sonstiges" (english "Other") only appear if syncting is running. I tested how syncthing work if the Network is blocked by AFWall. Even syncthing is blocked the "Other" app consumes around 20%. Power.

**Comment by Nutomic** [comment]
What do the logs show?

**Comment by fti7** [comment]
Similar here - Nexus5X with Android O - 8.0
It Shows ~20% directly for the Syncthing App during a Work-Day.

The Mainlog doesnt show very much (reconnects etc). 
The Android Log is full with "creating folder observer" entries... but i guess thats normal.

Any Debug Flag to add to dig deeper?

BR,
Frank

**Comment by Nutomic** [comment]
@calmh @AudriusButkevicius Any idea how to debug this?

**Comment by AudriusButkevicius** [comment]
It's not optimized for mobiles so it probably keeps the cpu awake and tries to reconnect indefinately.

**Comment by Mannshoch** [comment]
[Log-2017-09-25-06.07.45.txt](https://github.com/syncthing/syncthing-android/files/1330135/Log-2017-09-25-06.07.45.txt)

Should I open a New Ticket on syncthing/syncthing ?

**Comment by AudriusButkevicius** [comment]
This is something else, it seems the issue with the wrapper trying to launch/talk to the binary indefinately.

**Comment by Nutomic** [comment]
@Mannshoch What were you doing when you recorded this log? Did you have the app open or in the background? I assume the app was disabled due to wifi/charging conditions?

**Comment by Mannshoch** [comment]
I deactivated 3g/wlan. in this condition I opened sycthing. Background syncing is active. i switched to OS Monitor and exported the logs from syncthing.

**Comment by Nutomic** [comment]
So the app was disabled, right? From the logs, it seems like the app is waiting for Syncthing to launch.

**Comment by Mannshoch** [comment]
I don't understand. What you mean. Which app is waiting for Syncthing?

**Comment by Nutomic** [comment]
The Syncthing-Android app is waiting for the native Syncthing binary to start.

Anyway can you provide steps to reproduce this behaviour? I couldnt reproduce it on my device.

**Comment by Mannshoch** [comment]
I will do it if I know what I did to get that.
I have three devices and two directories, background settings and Android battery settings active

**Comment by Catfriend1** [comment]
@Mannshoch I think this got solved with version 0.10.8. Does that upgrade help?

**Comment by AudriusButkevicius** [comment]
I am just closing this as this is ancient.
